### DISCUSSION TITLE
Editor Team Question: Pausing and Unpausing

### DISCUSSION THREAD

**Discussion by minecraftchloe** [Mojang/minecraft-editor#7] [Q&A]
What would be your ideal experience for pausing actors or blocks in a single or multi(creator) experience?

**Comment by JWForever5504** [comment]
I've wanted this feature since I found out about developer commands. It is so cool that we can have it now, even if not in regular worlds.

My wish is that all players in a world have to agree to pause or unpause the game, in case someone is doing something where unpausing world break their build.

Thank you for all that you have done! This is just so amazing!

**Comment by tryashtar** [comment]
My dream is being able to "snapshot" the world while it's paused.
You click a button, and the current state of the world is saved. Then you can make any changes you want, even resuming the action and testing it yourself in survival mode. If you make a change you don't like, or finish your test, you can restore from your snapshot to go right back to where you were before.
